furusiya/model/entities/party/player.py

```diff
@@ -49,13 +49,22 @@ def unmount(self, horse):
             horse.is_mounted = False
 
     def rest(self):
-        if config.data.features.allowResting:
-            fighter = self.get_component(Fighter)
-            hp_gained = int(config.data.player.restingRatio * fighter.max_hp / fighter.hp)
-            fighter.heal(hp_gained)
-            return 'rested'
-        else:
-            return 'didnt-take-turn'
+        fighter = self.get_component(Fighter)
+        hp_gained = int(config.data.player.restingPercent * fighter.max_hp / fighter.hp)
+        fighter.heal(hp_gained)
+        return 'rested'
+
+    def restTillFull(self):
+        fighter = self.get_component(Fighter)
+        temp_health = fighter.hp
```

Comment:
I think you can get rid of `temp_health`

Reply:
I think not; we're not actually changing health here, we're just calculating how many turns it takes to rest to full health.